README.md

```diff
@@ -1,31 +1,104 @@
+|     CI      | [![Build Docker Images](https://github.com/Quansight/qhub/actions/workflows/image.yaml/badge.svg)](https://github.com/Quansight/qhub/actions/workflows/image.yaml) [![Kubernetes Tests](https://github.com/Quansight/qhub/actions/workflows/kubernetes_test.yaml/badge.svg)](https://github.com/Quansight/qhub/actions/workflows/kubernetes_test.yaml) [![Tests](https://github.com/Quansight/qhub/actions/workflows/test.yaml/badge.svg)](https://github.com/Quansight/qhub/actions/workflows/test.yaml) |
+| Website | [![Website Deploy](https://github.com/nebari-dev/nebari/actions/workflows/deploy.yml/badge.svg)](https://github.com/nebari-dev/nebari/actions/workflows/deploy.yml) 
 
-🏗 An opinionated open source JupyterHub distribution
+---
 
-## Documentation
+Nebari is an customizable Data Science and MLOPs Platform designed as a managed integration of open source technologies. It uses an infrastructure-as-code approach to quickly and easily deploy a shared data science environment, on premises or in the cloud of choice. 
 
-### Installing  `node`
-- Checking if node is already present in the system (should print the version number)
-    ```bash
-     node -v
-    ```
-- To install node visit [nodejs.org](https://nodejs.org/en/download/)
+Nebari is an opinionated open source JupyterHub distribution with a variety of common integrations to help you and your team get set up with all your favorite tools ...  and a few you didn't know you needed! 
 
-### Building the site
+Check out our full [documentation](https://nebari-docs.netlify.app/)!
 
-Install the neccesary packages
-```bash
-cd docs
-npm install
-```
+---
+
+## Table of contents
+
+- [Nebari 101](#nebari-101)
+- [Nebari on-prem](#nebari-on-prem)
+  <!-- - [:cloud: Cloud Providers](#cloud-cloud-providers) -->
+- [:computer: Installation](#computer-installation)
+- [:question: Questions?](#question-questions)
+- [:book: Code of Conduct](#book-code-of-conduct)
+- [:gear: Installing the Development version of Nebari](#gear-installing-the-development-version-of-nebari)
+- [:raised_hands: Contributions](#raised_hands-contributions)
+- [License](#license)
+
+## Nebari 101
+
+Nebari Cloud deployments use Kubernetes and are is built using [Terraform](https://www.terraform.io/), [Helm](https://helm.sh/), and
+[GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions). Terraform handles the build, change, and versioning of the infrastructure. Helm helps to define, install,
+and manage [Kubernetes](https://kubernetes.io/ "Automated container deployment, scaling, and management"). GitHub Actions is used to automatically create commits when the
+configuration file (`nebari-config.yaml`) is rendered, as well as to kick off the deployment action.
```

Comment:
`nebari-config.yaml` - not sure if this will still be a thing? should we just update later?

Reply:
I think this one is worth mentioned in the dev-channel as well

Reply:
always find this confusing - what do we mean by `config.yml` rendering?